src/components/custom/dashboard/client/pages/diet-tracker/CreateFoodTab.jsx

```
@@ -0,0 +1,359 @@
+            placeholder="e.g., Grilled Chicken Salad, Apple, Protein Shake"
+            error={errors.name}
+            required
+            className="w-full"
+          />
+
+          {/* Snack Type Selection - Only show if requested */}
+          {showMealTypeSelector && (
+            <div className="space-y-2">
+              <label className="block text-sm font-medium text-white">
+                Type
+              </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)